src/pages/BoardPage/BoardRegisterPage/BoardRegisterPage.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useRef, useState } from "react";
+import styled from "styled-components";
+
+import camera from "@/assets/Board/camera.svg";
+import gallery from "@/assets/Board/gallery.svg";
+import link from "@/assets/Board/link.svg";
+import CheckBox from "@/components/CheckBox";
+import TopBarText, { LeftEnum } from "@/components/TopBarText";
+
+const BoardRegisterBtn = styled.button`
+  color: var(--Foundation-Gray-gray500, #767681);
+  /* text/Regular 16pt */
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 140%; /* 1.4rem */
+  letter-spacing: 0.04rem;
+
+  .board-register-btn-active {
+    color: var(--Foundation-Gray-white, #fff);
+  }
+`;
+
+const BoardRegisterContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: 0 1.25rem;
+`;
+
+const BoardRegisterManagerTitle = styled.div`
+  width: 100%;
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  padding: 0.5rem 0;
+  margin-top: 1rem;
+  border-bottom: 1px solid #222226;
+
+  color: var(--Foundation-Gray-gray600, #45454b);
+  font-size: 0.875rem;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 140%;
+
+  .board-register-manager-active {
+    color: var(--Foundation-Main-color-Normal, #48ffbd);
+  }
+`;
+
+const BoardRegisterTitle = styled.input`
+  margin-top: 1.5rem;
+  width: 100%;
+  padding-bottom: 0.75rem;
+  border: none;
+  border-bottom: 1px solid #222226;
+  background: none;
+
+  color: var(--Foundation-Gray-white, #fff);
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 140%; /* 1.4rem */
+
+  &::placeholder {
+    color: var(--Foundation-Gray-gray400, #acacb5);
+  }
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const BoardRegisterContent = styled.textarea`
+  margin-top: 1rem;
+  width: 100%;
+  height: auto;
+  border: none;
+  background: none;
+
+  color: var(--Foundation-Gray-white, #fff);
+  /* regular/14pt */
+  font-size: 0.875rem;
```

Comment:
textarea 내부에서 폰트가 안 바뀌네요!ㅠ

```suggestion
  font-family: Freesentation;
  font-size: 0.875rem;
```

Reply:
반영했습니닷